src/components/CameraControls/index.tsx

```diff
@@ -189,7 +189,7 @@ const CameraControls = ({
                                 className={classNames([
                                     "icon-moon",
                                     "anticon",
-                                    styles.rotate,
+                                    "rotate-icon",
```

Comment:
This points to global style rule, which references css variable, rather than pointing at component style sheet that directly references glyph code in content property.